dist/index.js

```diff
@@ -0,0 +1,215 @@
+
+function _possibleConstructorReturn(self, call) { if (!self) { throw new ReferenceError("this hasn't been initialised - super() hasn't been called"); } return call && (typeof call === "object" || typeof call === "function") ? call : self; }
+
+function _inherits(subClass, superClass) { if (typeof superClass !== "function" && superClass !== null) { throw new TypeError("Super expression must either be null or a function, not " + typeof superClass); } subClass.prototype = Object.create(superClass && superClass.prototype, { constructor: { value: subClass, enumerable: false, writable: true, configurable: true } }); if (superClass) Object.setPrototypeOf ? Object.setPrototypeOf(subClass, superClass) : subClass.__proto__ = superClass; } /**
+                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               * @providesModule Hyperlink
+                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               **/
+
+var textPropTypes = _reactNative.Text.propTypes || {};
+var OS = _reactNative.Platform.OS;
+
+var Hyperlink = function (_Component) {
+  _inherits(Hyperlink, _Component);
+
+  function Hyperlink(props) {
+    _classCallCheck(this, Hyperlink);
+
+    var _this = _possibleConstructorReturn(this, (Hyperlink.__proto__ || Object.getPrototypeOf(Hyperlink)).call(this, props));
+
+    _this.linkify = _this.linkify.bind(_this);
+    _this.parse = _this.parse.bind(_this);
+    _this.linkifyIt = props.linkify || require('linkify-it')();
+    return _this;
+  }
+
+  _createClass(Hyperlink, [{
+    key: 'componentWillReceiveProps',
+    value: function componentWillReceiveProps() {
+      var _ref = arguments.length > 0 && arguments[0] !== undefined ? arguments[0] : {},
+          _ref$linkify = _ref.linkify,
+          linkify = _ref$linkify === undefined ? require('linkify-it')() : _ref$linkify;
+
+      this.linkifyIt = linkify;
+    }
+  }, {
+    key: 'render',
+    value: function render() {
+      var viewProps = _objectWithoutProperties(this.props, []);
+
+      delete viewProps.onPress;
+      delete viewProps.linkDefault;
+      delete viewProps.onLongPress;
+      delete viewProps.linkStyle;
```

Comment:
did these props cause any problems?

Reply:
Yeah they did. Mostly warnings with react-native-web. They were being passed to a view and views don't contain those props. So you would get prop warnings.

Reply:
ah ok. thanks @rardoz